packages/components/src/Components/Dark/Dark.d.ts

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
```

Comment:
I think all these declaration type files are here by mistake, aren’t they?

Reply:
I'm not entirely sure tbh, haven't touched these files myself.

Reply:
Yep, then it should not be here too.